configurations/audit.yaml

```diff
@@ -0,0 +1,5 @@
+append_scylla_yaml: |
+  audit: "table"
+  audit_categories: "DCL,DDL,AUTH,ADMIN"
+  audit_tables: "keyspace1.standard1,scylla_bench.test"
+  audit_keyspaces: "keyspace1,scylla_bench"
```

Comment:
What will be the impact of auditing `keyspace1` for regular tests?
Both disk usage amplification and performance wise, as it may turn things slower and disruptions maybe can timeout because of it.
was this tested?

Reply:
I will update the byo test status in another PR: https://github.com/scylladb/scylla-cluster-tests/pull/3090#discussion_r551695749